test/e2e/alertmanager_test.go

```diff
@@ -231,3 +226,187 @@ receivers:
 		t.Fatal(err)
 	}
 }
+
+func TestAlertmanagerZeroDowntimeRollingDeployment(t *testing.T) {
+	t.Parallel()
+
+	ctx := framework.NewTestCtx(t)
+	defer ctx.Cleanup(t)
+	ns := ctx.CreateNamespace(t, framework.KubeClient)
```

Comment:
@brancz Can you add the PrometheusRBAC setup just like I did [here](https://github.com/coreos/prometheus-operator/blob/master/test/e2e/prometheus_test.go#L46)? Switching to the tectonic-installer forces us to care about RBAC initialization.